src/components/header/index.js

```diff
@@ -21,18 +21,18 @@ export const Header = () => {
     <>
       <div className={styles.container}>
         <div className={styles.content}>
-          <Button onClick={() => handleRoute('/')}>
+          <Button onClick={() => handleRoute('/')} aria-label="Back to home">
```

Comment:
This is super important, but I'm not sure I like the API. mainly because the button isn't rendering the injected props. so this will not be attached. You need to add {...props} on the `return ( <Button {...props}>` something like that.

Reply:
Fixed this. The prop is quite long now (ariaLabel) but it could be something like a11y. However that doesn't really say much.